src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -250,6 +252,7 @@ export default class MultipleProjectSelector extends React.PureComponent {
                   this.handleClear();
                   actions.close();
                 }}
+                message={footerMessage}
```

Comment:
Would be good to have a percy snapshot or jest test for this. The multi-project selector is quite complicated and prone to breaking.

Reply:
Agreed, I would like to make a percy snapshot of this state once everything on releases v2 page is stable. I am writing it down so I won't forget 👍